src/test/java/net/masterthought/cucumber/generators/AbstractPageTest.java

```diff
@@ -48,8 +43,32 @@ public void generateReport_CreatesReportFile() {
         assertThat(reportFile).exists();
     }
 
+
+    @Test
+    public void generateReport_DisplaysContentAsEscapedText() {
+        setUpWithJson(HTML_JSON);
+
+        // given
+        page = new FeatureReportPage(reportResult, configuration, features.get(0));
+
+        // when
+        page.generatePage();
+
+        // then
+        DocumentAssertion document = documentFrom(page.getWebPage());
+        assertThat(document.getFeature().getDescription())
+                .isEqualTo("<b>a bold statement</b>");
+        assertThat(document.getFeature().getElements()[0].getDescription())
+                .isEqualTo("<b>a bold statement</b>");
```

Comment:
name of the method says that data should be escaped while tests validates against not-escaped content - can you tell me more about it?